Frameworks/MQTTnet.AspnetCore/AspNetMqttServerOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using MQTTnet.Server;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MQTTnet.AspNetCore
+{
+    public class AspNetMqttServerOptions : MqttServerOptions, IAspNetMqttServerOptions
+    {
+        public Boolean ListenTcp { get; set; } = true;
+    }
```

Comment:
There is already a option for this in the MqttServerOptions.

Reply:
It's actually an option on MqttServerDefaultEndpointOptions and MqttServerTlsEndpointOptions
I was trying to be more explicit and user friendly on the helper methods. Disabling the tcp adapter is clearly an option that only applies when others are in use.